storage/tests/unit/test_bucket.py

```diff
@@ -197,8 +197,8 @@ def test_ctor_defaults(self):
         config = self._make_one(bucket)
 
         self.assertIs(config.bucket, bucket)
-        self.assertFalse(config.bucket_policy_only_enabled)
-        self.assertIsNone(config.bucket_policy_only_locked_time)
+        self.assertFalse(config.uniform_bucket_level_access_enabled)
+        self.assertIsNone(config.uniform_bucket_level_access_locked_time)
```

Comment:
We need to retain tests for the deprecated properties (which would have exposed the "can't call properties" bugs I noted above).